### ISSUE TITLE
Document using the overrides with lima

### ISSUE THREAD

**Issue by mattfarina** [rancher-sandbox/rancher-desktop#1343]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://github.com/rancher-sandbox/rancher-desktop/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

People are asking about changing some lima settings. We have had to tell people how to using the lima overrides to route around issues.

### Proposed Solution

Document the overrides along with a notice of caution about how this can create issues with future releases of RD and lima.

**Comment by jtravee** [comment]
cc @jandubois

**Comment by scalp42** [comment]
It seems that placing an `override.yaml` in `~/Library/Application Support/rancher-desktop/lima/_config` doesn't work as the directory gets wiped/reset by Rancher.

**Comment by jandubois** [comment]
> the directory gets wiped/reset by Rancher.

This should only happen during a Factory Reset, which indeed wipes everything (except the update cache, which is a bug).

**Comment by scalp42** [comment]
Gotcha, I tried creating an `override.yaml`  in `~/Library/Application Support/rancher-desktop/lima/_config` with:

```yaml
ssh:
  loadDotSSHPubKeys: true
  forwardAgent: true
```

Then tried to restart the Lima VM:

```
LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima" "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl" stop 0

LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima" "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl" start 0
```

But then any Docker command fails:

```
$> docker ps
error during connect: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json": EOF
```

Not too sure

Related: https://github.com/rancher-sandbox/rancher-desktop/discussions/1842

**Comment by jandubois** [comment]
> Then tried to restart the Lima VM:

You should not manually restart the VM; you should stop Rancher Desktop, and then start it again.

I'm not sure if this the cause of your issue with the docker socket, but it is the first thing I would try: let RD restart the VM itself.

**Comment by scalp42** [comment]
Indeed, doing by hand is not recommended (needed to restart Docker daemon). Thanks for the quick help. I'll move to the discussion above regarding ssh-agent issues.
